src/v2/Apps/Order/Components/PaymentPicker.tsx

```diff
@@ -42,6 +40,8 @@ import {
 } from "v2/System/SystemContext"
 import { createStripeWrapper } from "v2/Utils/createStripeWrapper"
 import { isNull, mergeWith } from "lodash"
+import { BillingInfoFormContext } from "v2/Apps/Auction/Components/Form"
```

Comment:
I guess the Form Context is in the wrong place, if it's a shared thing across different apps should not be inside Auction